content/_index/coffeeeditorfeatures/modelanalysis.md

```diff
@@ -0,0 +1,9 @@
++++
+title = "Model Analysis and Chart Visualization"
+weight = 50
+
+[asset]
+  icon = "fas fa-chart-pie"
++++
+
+The coffee editor provides an example model analysis, the result is visualized as a "sun burst" chart. The analysis is an external component written in Kotlin, the chart is based on D3. Select the file "superbrewer3000.wfconfig" in the coffee editor, press F1, type "Analyze workflow model" and hit enter to see the model analysis in action!
```

Comment:
you can also right click inside the editor to call this action

Reply:
Which editor?

Reply:
the .wfconfig editor

Reply:
Hmm shall we really mention this? It breaks the layout, as the text gets longer :-)

Reply:
I will mention it in the welcome page

Reply:
It was just a pointer, I don't care what you do with this information ;-)